src/routes/+page.svelte

```diff
@@ -188,4 +192,8 @@
 			opacity: 1;
 		}
 	}
+
+	.btn:focus {
+		outline: none !important;
+	}
```

Comment:
it's usually good to have some sort of indicator of focused elements ... 
was the outline too ugly?

Reply:
When I opened the modal, there were 2 blue borders around random elements by default. It kinda bothered me so I'm not sure what to do... (It's a blue border around the insta button and the bottom left corner of the yellow div)

Reply:
oh maybe it's autofocusing random elements in the modal on show 
do you want to try moving this css style to the Modal component you made if the issue was specifically for the modal?
it's generally not great to remove all focus indicators for a whole page

Reply:
you can also use `outline-none` tailwind class to remove outlines!
https://tailwindcss.com/docs/outline-style

Reply:
Fixed it! I thought it was the buttons but it was the <a> tags that was causing it